js/main.js

```diff
@@ -24,3 +41,42 @@ function checkLengthLine(textLine, maxLength) {
 }
 
 checkLengthLine('Hello', 5);
+
+function createComment (commentNumber) {
+  return {
+    id: commentNumber,
+    avatar: `img/avatar-${getRandomInteger(MIN_COUNT_AVATAR, MAX_COUNT_AVATAR)}.svg`,
+    message: COMMENTS_USERS[getRandomInteger(0, COMMENTS_USERS.length - 1)],
+    name: NAMES_USERS[getRandomInteger(0, NAMES_USERS.length - 1)],
+  };
+}
+
+function createComments () {
+  const countComments = getRandomInteger(MIN_COMMENT_COUNT, MAX_COMMENT_COUNT);
+  const comments = new Array(countComments).fill(null);
+  for (let i = 0; i < countComments; i++) {
+    comments[i] = createComment(i+1);
+  }
+  return comments;
```

Comment:
```suggestion
  const comments = new Array(countComments).fill(null);
  
  for (let i = 0; i < countComments; i++) {
    comments[i] = createComment(i+1);
  }
  
  return comments;
```

можно тут пустые строки вставлять, чтобы код был более читаемым